README.md

```diff
@@ -6,10 +6,29 @@ Respond to short response questions in clear, concise sentences directly within
 ### Short Response Questions
 **1. What is recursion?**
 
+    Recursion is the concept of a function calling upon itself in their function body.
```

Comment:
:+1: Really thorough. Looking for more on the base case and what must happen to the function calls in the recursive case